Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:bullseye-slim as luarocks-installer
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+      build-essential \
+      libpcre2-dev \
+      libssl-dev \
+      libxml2 \
+      lua5.1 \
+      luarocks && \
+    luarocks install --tree /luarocks luaossl && \
+    luarocks install --tree /luarocks lrexlib-pcre2 && \
+    luarocks install --tree /luarocks luautf8 && \
+    luarocks install --tree /luarocks xmlua && \
+    luarocks install --tree /luarocks base64
+
+FROM fluent/fluent-bit:1.9-debug
```

Comment:
@patrick-stephens if I change this to `fluent/fluent-bit:1.9` things stop working. Any clues why?

Reply:
It seems to build and run for me fine, I'm not using LUA stuff so do you mean something there does not work?

My guess is there are some extra dependencies we need to add that are not just the luarocks ones.

Reply:
Ah yeah, you need to split that out like we do here: https://github.com/fluent/fluent-bit/blob/0c41d2bf1a63a1b83a0323b2c87024537cbaad59/dockerfiles/Dockerfile#L101-L139

Reply:
Done, I used a different method to extract the libraries, but hopefully this will be unnecessary in the future if we ship a standard library for lua filters.